doc/cross-compilation.xml

```diff
@@ -181,7 +181,7 @@
       How does this work in practice? Nixpkgs is now structured so that build-time dependencies are taken from <varname>buildPackages</varname>, whereas run-time dependencies are taken from the top level attribute set.
       For example, <varname>buildPackages.gcc</varname> should be used at build time, while <varname>gcc</varname> should be used at run time.
       Now, for most of Nixpkgs's history, there was no <varname>buildPackages</varname>, and most packages have not been refactored to use it explicitly.
-      Instead, one can use the four attributes used for specifying dependencies as documented in <xref linkend="ssec-stdenv-attributes"/>.
+      Instead, one can use the five attributes used for specifying dependencies as documented in <xref linkend="ssec-stdenv-dependencies"/>.
```

Comment:
If you like overengineering, feel free to write a script that does the count for you and injects it.